src/main/scala/io/iohk/ethereum/vm/Int256.scala

```diff
@@ -0,0 +1,201 @@
+package io.iohk.ethereum.vm
```

Comment:
@mcsherrylabs @whysoserious on the branch for EC-85, that @AlanVerbner is working on, `DataWord` escapes the `vm` package and is used in transaction execution. That's one more reason to put it in `util` package.